### DISCUSSION TITLE
Tutorial Video request list

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#2892] [Ideas]
We have currently reached a development state where i might have some time left to do tutorial videos again. 

The quickstart videos are still working fine for Bforartists 3. So i don't want to redo them at the moment just for the sake of having two or three new things in the UI displayed. Also, keep in mind that most Blender tutorials works just fine in Bforartists.

So, what are your thoughts, what are your wishes? What tutorial videos do you want to see here? What kind of tutorials should we create?

**Comment by Draise14** [comment]
Blender vs Bforartists workflows:
- the different modes and editors comparisons (highlighting the pros of Bforartists l
- the different customizations you can do to refine your workflow with the toolbar and toolshelf
- default hotkey differences
- tooltip differences
- documentation differences
- default preferences (load UI etc)


Bforartists feature how tos:
- how to use the node toolshelf
- how to use the quick editor switches
- how to use the 3D toolshelf
- How to use the toolbar
- How to use the custom overlay buttons
- how to use the hotkey overlays and use the default hotleys

Blender compatibility:
- how to translate a blender tutorial for Bforartists
- how to troubleshoot an addon
- How to download the dev builds to test addon compatibility

Development and feedback
- how to report a feature request
- how to report discussions
- how to use discord
- how to use the forums
- how to become a developer
- how to use branches, pull requests and to commit

**Comment by ReinerBforartists** [comment]
Hm, i thought more of learning videos ^^

**Reply by Draise14** [reply]
Yeah. Learning videos could be useful but we have all the Blender community for that already. We have a software with a GUI philosophy over Blender that needs to be known and supported, those bases could be covered. 

The other idea is this:
How to make an art piece from 0 to final render etc. Micro courses. 
- game asset prop
- realistic environment with a lighthouse
- animatable character
- a car
- a product showcase animation

**Reply by ReinerBforartists** [reply]
... and a donut maybe? :D

Good point. I will have a look if i can come up with a UI related tutorial.

**Comment by ReinerBforartists** [comment]
Assigning to the tooltips now. Let's have a look if i can come up with an interesting video.

EDIT, tooltip video done https://www.youtube.com/watch?v=wT57fv9MWrU

**Comment by ReinerBforartists** [comment]
**the different modes and editors comparisons (highlighting the pros of Bforartists l**

We have this here already. The differences between Blender 3 and Bforartist 3: https://www.youtube.com/watch?v=iXu2t3e9NkA

**the different customizations you can do to refine your workflow with the toolbar and toolshelf**
Toolbar is explained in the quickstart videos. This is bfa 1 though. So we might want to redo it. But it is still valid: https://www.youtube.com/watch?v=6aSZTKeKWkI

Toolshelf video may be missing. But it is the same content than in the text menus. Feel free to make a dedicated video for it. I don't see a need for.

**Comment by ReinerBforartists** [comment]
Super important: how to deal with blender tutorials. Seems that i have to record a Blender tutorial too :D

Edit. Done: https://www.youtube.com/watch?v=i6_Cqi_WCwY
